iModelCore/iModelPlatform/DgnCore/DgnUnits.cpp

```diff
@@ -5,6 +5,26 @@
 #include <DgnPlatformInternal.h>
 #include <DgnPlatform/DgnGeoCoord.h>
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+StatusInt EcefLocation::ComputeTransform(TransformR ecefTrans)
+    {

```

Comment:
```suggestion
    {
    if (!m_isValid)
        return Transform::FromIdentity();

```
The other members are uninitialized garbage if `m_isValid` is `false`.